content/How to create an Activity Indicator with SwiftUI - Pablo Malvasio.md

```diff
@@ -0,0 +1,210 @@
+![Main image](images/spinner-cover.jpg)
+
+# How to create an Activity Indicator with SwiftUI
+
+## Introduction
+
+The year was 2019 (far far away from any COVID-19 or natural disasters) and me as many other developers were at the San Jose McEnery Convention Center waiting for all the new announcements Craig, Tim and all the engineers from Apple were about to share with us.  
+
+Thats's when SwiftUI took us by storm. We’re delighted by this new way to create our UI on a declarative way and let the data flow between the views with all the new tools SwiftUI and Combine provide. This new addition to the Apple family was huge, and even that was too early for production, we couldn’t stop ourselves to start playing around with it. But this "beta" state started to being noticed when some performance issues or even lack of support for some primitive components that were part of our daily work appeared. 
```

Comment:
`even that was too early for production` sounds weird to me maybe: 
`even though it wasn't production-ready`?

Reply:
`However,` this "beta"...

Reply:
👍